spec/features/commands_spec.rb

```diff
@@ -157,6 +157,18 @@
     end.to change { deployment_requests.count }.by(1)
   end
 
+  scenario 'checking a locked branch' do
+    # It says that it's locked after someone locks it.
+    command "/deploy lock staging on acme-inc/api: I'm testing some stuff", as: slack_accounts(:david)
+    command '/deploy check lock staging on acme-inc/api', as: slack_accounts(:david)
```

Comment:
It does, but since environments are currently created just-in-time, there's a lot of bogus environments from typos. Something that could be cleaned up later.